CRUDRecipeEF.BL.DL/Data/DataSeed.cs

```diff
@@ -1,40 +1,54 @@
 ﻿using System.Collections.Generic;
-using CRUDRecipeEF.BL.DL.Entities;
+using CRUDRecipeEF.BL.DL.DTOs;
+using CRUDRecipeEF.BL.DL.Services;
+using System.Linq;
 
 namespace CRUDRecipeEF.BL.DL.Data
 {
-    public static class DataSeed
+    public class DataSeed : IDataSeed
     {
-        public static void Seed(RecipeContext context)
```

Comment:
Whats the reason for changing the whole functionality of the class?

Reply:
When retrieving recipes they had no ingredients attached even though they were present in the DB